docs/how-to-maintain.rst

```diff
@@ -0,0 +1,113 @@
+How To Maintain
+===============
+
+This document is dedicated to maintainers of GETTSIM.
+
+
+Versioning
+----------
+
+GETTSIM adheres in large parts to `semantic versioning <https://semver.org>`_. Thus, for
+a given version number ``major.minor.patch``
+
+* ``major`` is incremented when you make incompatible API changes.
+* ``minor`` is incremented when you add functionality which is backwards compatible.
+* ``patch`` is incremented when you make backwards compatible bug fixes.
+
+You can add a suffix to signal more information on the current version.
+
+* Use ``0.0.1dev0`` to signal that the release is a development version which is
+  currently tested and to be released soon. Increment the last digit for multiple
+  development versions.
+
+.. note::
+
+   The development suffix is not separated with a dot from the patch version as this
+   does not comply with the publishing process on anaconda.org.*
+
+
+Branching Model
+---------------
+
+The branching model for GETTSIM is very simple.
+
+1. The master branch is the target of ongoing development. Every PR should be merged
+   into master.
+
+   (The only deviation can happen with respect to backports. See :ref:`backports` for
+   more.)
+
+2. There can exist branches for minor versions called, for example, ``0.1.``. The
+   branches contain changes, e.g. backports, which are not released yet. To find out how
+   to release the next patch version, see the following section. If there are no
+   additional changes waiting to be released, the branches should be deleted for
+   clarity.
+
+
+.. _releases:
+
+How To Release
+--------------
+
+To release a new version of GETTSIM, do the following.
+
+1. To start a new release for version ``0.2.1``, create a new milestone with the version
+   as its name on Github to collect issues and PRs which should be included in the new
+   version.
+
+2. Call a halt on changes to the master branch. Make sure all changes to the master
+   branch since the last version are included in the milestone.
+
+**Here, we have to decide whether we allows pushing to master for releases or not. Then,
```

Comment:
It does not make sense to me anymore either. It was related to the fact that the master branch is in my projects protected so that only PR merges are allowed, but it does not make sense anymore.